db/dataset.py

```diff
@@ -315,3 +315,27 @@ def _delete_snapshots_for_dataset(connection, dataset_id):
         DELETE FROM dataset_snapshot
               WHERE dataset_id = :dataset_id""")
     connection.execute(query, {"dataset_id": dataset_id})
+
+
+def add_recordings(dataset, dataset_id):
+    """Adds new recordings to a class in a dataset"""
+
+    with db.engine.begin() as connection:
+        result = connection.execute("""SELECT id FROM dataset_class WHERE name = %s and dataset = %s""",
+                                    (dataset["class_name"], dataset_id))
+        clsid = result.fetchone()
+
+        for mbid in dataset["recordings"]:
```

Comment:
If there are multiple recording ids in the json list and one of them is already there in the class  in database, do we raise an error without inserting any of the recording ids, or should we modify the json by removing the incorrect (duplicate) recording id and going ahead by adding rest of them? Also, is a scenario possible where the duplicate recording ids are present within the json list itself? 
For example, can we have a json like the following:
{
            "class_name": "Happy",
            "recordings": ["770cc467-8dde-4d22-bc4c-a42f91e", "770cc467-8dde-4d22-bc4c-a42f91e"]
        }
Do we need to put a check for this as well?